packages/calcite-components/src/components/sort-handle/sort-handle.tsx

```diff
@@ -136,13 +136,13 @@ export class SortHandle extends LitElement implements InteractiveComponent {
   calciteSortHandleClose = createEvent({ cancelable: false });
 
   /** Fires when a move item has been selected. */
```

Comment:
Side note: we should add a doc snippet for cancelable events. cc @geospatialem @DitwanP